thirdparty/README.md

```diff
@@ -1,5 +1,12 @@
 # Third party libraries
 
+## assimp
```

Comment:
Keep two newlines between each section, as done for the other ones.

Reply:
Done.